lib/active_admin/filters/active_filter.rb

```diff
@@ -26,13 +26,21 @@ def values
       end
 
       def label
+        translated_predicate = predicate_name.mb_chars.downcase.to_s
         if related_class
-          "#{related_class.model_name.human} #{predicate_name}".strip
+          "#{related_class.model_name.human} #{translated_predicate}".strip
         else
-          "#{attribute_name} #{predicate_name}".strip
+          "#{attribute_name} #{translated_predicate}".strip
         end
       end
 
+      # translated active admins predicated equals, contains, starts_with ..
+      # with fallback to ransack eq, cont, start ...
+      def predicate_name
+        key = condition.predicate.name
+        I18n.t("active_admin.filters.predicates.#{key}", default: Ransack::Translate.predicate(key))
+      end
```

Comment:
@Fivell I've learned over time that while comments aren't a bad thing they are failing us here because its telling us the code isn't as descriptive/readable as it could be. We can avoid the comments by just breaking this out with descriptive methods that make it clear what we want to do. For example, something like:

```ruby
def translated_predicate_name
  I18n.t("active_admin.filters.predicates.#{condition.predicate.name}", 
    default: fallback_to_ransack_predicate_name)
end

def fallback_to_ransack_predicate_name
  Ransack::Translate.predicate(condition.predicate.name)
end
```

Which also has the nice benefit of avoiding the local variable too.